### ISSUE TITLE
Rule: Comma spacing [$15 awarded]

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#628]
This rule should allow you to specify spacing for commas, as in, some combination of spaces before and after. The options should look like:

```
comma-spacing: [2, { before: false, after: true }]
```

(These should also be the default settings.)

It should flag when the spacing is unexpected and be on by default.
##
